src/xrGame/DiscordRPC.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+class DiscordRPC
+{
+private:
+    int64_t     start_time          {};
+    string64    build_name          {};
+    string64    current_level_name  {};
+    pcstr       current_task_name   {};
+
+public:
+    DiscordRPC      (){ start_time = time(nullptr); };
+    ~DiscordRPC     (){};
```

Comment:
Move constructor to .cpp
```suggestion
    DiscordRPC() { start_time = time(nullptr); }
    ~DiscordRPC();
```

Reply:
And make destructor default. There's no point in making it empty.

Reply:
Made `~DiscordRPC() = default;` 'cause compiler complains on `~DiscordRPC();`